x/tokenfactory/README.md

```diff
@@ -45,7 +45,7 @@ message MsgCreateDenom {
 ### Mint
 
 Minting of a specific denom is only allowed for the creator of the denom
-registered during `CreateDenom`.
+registered during `CreateDenom`, or the current admin.
```

Comment:
These updates should all probably be:
```
Minting of a specific denom is only allowed for the current admin (default creator of the denom)
```

(Since its not the case that its denom creator AND current admin)